### PR TITLE
Fixed internal error found by OpenQA

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: src/modules/LanItems.rb [hunk 1]
```diff
@@ -47,6 +47,8 @@ class LanItemsClass < Module
 
     include Logger
 
+    require "network/install_inf_convertor"
```
Comment (jreidinger) [inline src/modules/LanItems.rb]: require should be top level call unless really needed.

Comment (jreidinger) [inline src/modules/LanItems.rb]: so below line 25 I will place it

**Comment by jreidinger** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for review

**Comment by kobliha** [comment]
@mchf Hmm and does it mean that we DO NOT have a unit test for the new code?

**Comment by jreidinger** [comment]
@kobliha as discussed on IRC problem is that mass execution of tests in rspec preload file earlier. I think tricks with unloading class is not necessary, as it is one time fix and should not break easily. BTW running test standalone will discover it.

**Comment by mvidner** [comment]
@jreidinger Hmm! We could solve this by running the tests twice:
1. without coverage measurement, and without the preloading, to catch missing imports. 
2. with coverage measurement

**Pull request merged**